### PR TITLE
Animals table

### PR DESCRIPTION
**Pull request by why-a-en**
# Branch changes
- Created animals table with the columns provided
- Inserted the provided data
- Wrote queries for the quests mentioned in [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md)

- Screenshot 2023-06-06 233629
- Screenshot 2023-06-06 233711
- Screenshot 2023-06-06 233928
- Screenshot 2023-06-05 222319
- Screenshot 2023-06-06 233044
- Screenshot 2023-06-06 233109
- Screenshot 2023-06-06 233338
- Screenshot 2023-06-06 233551

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Alf, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented:
- Ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe. Similarly, kindly also remove guide notes for _license_ creation under the _license_ section.
- Ensure all headers have appropriate contents below describing them.
- The _Future Features_ section of your ReadMe includes 1-3 potential features to be added later. 
- Remove the Live demo link from the ReadMe if the app is not yet deployed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+select * from animals where RIGHT(nam
+e,3) = 'mon';
+
+select * from animals
```
Comment (Ismailco) [inline queries.sql]: - [x] You should select only animal names born between 2016 and 2019, instead yo're selecting all the info in the animal table
- [x] Also kindly make sure the second `date_of_birth` is `2019/31/12` as it is the last day of the year

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+where date_of_birth > '2016-01-01' and date_of_birth < '2019-01-01';
+
+select name from animals
```
Comment (Ismailco) [inline queries.sql:7]: - [x] Kindly consider selecting neurtred = 'true' explecitly

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @anbehindY, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (why-a-en) [reply queries.sql:7]: Could you pls explain why I need to do that explicitly?

File: README.md [hunk 3]
```diff
@@ -286,6 +106,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (Shedrack-Sunday) [inline README.md:134]: - [x] Kindly add a License file to the root of your project to stop it from breaking.
- Including a license file in a README is important as it clarifies the terms and conditions under which the software can be used, providing legal protection and promoting open collaboration while preventing potential misuse or infringement.

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @anbehindY , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 

You have submitted a project, that lacks the `Getting Started` section which has the `Setup` and `Usage` subsections

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @anbehindY , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Git flow is used correctly :white_check_mark: 
- [x] Animals table created correctly :white_check_mark: 
- [x] Pull request has a descriptive title and summary :white_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**